packages/core/state/selection/reducer.ts

```diff
@@ -147,10 +160,16 @@ export default makeReducer<SelectionStateBranch>(
             ...state,
             queries: action.payload,
         }),
-        [SET_SORT_COLUMN]: (state, action) => ({
-            ...state,
-            sortColumn: action.payload,
-        }),
+        [SET_SORT_COLUMN]: (state, action) => {
```

Comment:
A bit simpler
```
        [SET_SORT_COLUMN]: (state, action) => ({
            ...state,
            recentAnnotations: uniq([...castArray(action.payload?.annotationName), ...state.recentAnnotations]),
            sortColumn: action.payload,
        }),
```

Also I think you'll need the optional chaining check that this adds because `action.payload` could be undefined